  joint_state_publisher:
    type: joint_state_controller/JointStateController
    publish_rate: 50.0

# position controllers ---------------------------------------
  left_shoulder_controller:
    type: effort_controllers/JointPositionController
    joint: left_shoulder
    pid: {p: 10.0, i: 2.0, i_clamp: 1.0, d: 2.0}

  right_shoulder_controller:
    type: effort_controllers/JointPositionController
    joint: right_shoulder
    pid: {p: 10.0, i: 2.0, i_clamp: 1.0, d: 2.0}

  left_elbow_controller:
    type: effort_controllers/JointPositionController
    joint: left_elbow
    pid: {p: 10.0, i: 2.0, i_clamp: 1.0, d: 2.0}

  right_elbow_controller:
    type: effort_controllers/JointPositionController
    joint: right_elbow
    pid: {p: 10.0, i: 2.0, i_clamp: 1.0, d: 2.0}